include/swift/Basic/Features.def

```diff
@@ -229,6 +229,7 @@ EXPERIMENTAL_FEATURE(NamedOpaqueTypes, false)
 EXPERIMENTAL_FEATURE(FlowSensitiveConcurrencyCaptures, false)
 EXPERIMENTAL_FEATURE(CodeItemMacros, false)
 EXPERIMENTAL_FEATURE(PreambleMacros, false)
+EXPERIMENTAL_FEATURE(MacrosOnImports, true)
```

Comment:
This feature is missing a `usesFeatureMacrosOnImports()` function, is that intentional?

Reply:
No, it was a mistake! Should be fixed now

Reply:
From experience, it might be a good idea to expose whether `MacrosOnImports` is enabled as a `-D` flag in ClangImporter, allowing C headers to do feature detection. We do something similar for `__SWIFT_ATTR_SUPPORTS_SENDABLE_DECLS` (although that isn't tied to Swift language features—it's tied to an enabling clang change).

Reply:
Good idea! I went ahead and added it.